### PR TITLE
Add a recipe for ckan-editor-utils

### PR DESCRIPTION
**Pull request by ericmccowan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ckan-editor-utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
```
Comment (saraedum) [inline recipes/ckan-editor-utils/meta.yaml:21]:
```suggestion
    - python
```

File: recipes/ckan-editor-utils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - poetry >=1.1.3
+  run:
+    - boto3 >=1.15.16,<2.0.0
+    - python >=3.6,<4.0
```
Comment (saraedum) [inline recipes/ckan-editor-utils/meta.yaml:25]:
```suggestion
    - python
```

File: recipes/ckan-editor-utils/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+    - poetry >=1.1.3
```
Comment (saraedum) [inline recipes/ckan-editor-utils/meta.yaml:22]:
```suggestion
    - poetry >=1.1.3,<2
```

File: recipes/ckan-editor-utils/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  imports:
+    - ckan_editor_utils
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/ckan-editor-utils/
+  summary: Utilities for editing CKAN using its API.
```
Comment (saraedum) [inline recipes/ckan-editor-utils/meta.yaml:38]:
```suggestion
  summary: Utilities for editing CKAN, the Comprehensive Knowledge Archive Network
```

Just a suggestion. I had no idea what CKAN was and I find it nice to see acronyms expanded when I go to feedstocks without having to google them.

**Review by saraedum — CHANGES_REQUESTED** [review]
Looks good, though you don't need the pins on Python.

**Comment by ericmccowan** [comment]
Thank you for the suggestions, those have all been committed now.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**